test/e2e/network/kube_proxy.go

```diff
@@ -172,6 +172,8 @@ var _ = SIGDescribe("Network", func() {
 		}
 
 		jsonBytes, err := json.Marshal(options)
+		framework.ExpectNoError(err)
```

Comment:
please add annotation to this error - e.g.:
`framework.ExpectNoError(err, "could not marshal.....")`

Reply:
ok